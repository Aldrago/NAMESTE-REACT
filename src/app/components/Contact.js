const Contact = () => {
  return (
    <div>
      <h1 className="font-bold text-2xl">Contact us!</h1>
      <form>
        <input type="text" className="" placeholder="name" />
        <input type="text" className="" placeholder="message" />
        <button>Submit</button>
      </form>
    </div>
  );
};

export default Contact;
