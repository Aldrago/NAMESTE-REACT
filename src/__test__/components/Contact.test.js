import { render, screen } from "@testing-library/react";
import Contact from "../../app/components/Contact";
import "@testing-library/jest-dom";

describe("Contact us page test cases",() => {
  test("Should match snapshort", () => {
    const { container } = render(<Contact />);

    expect(container).toMatchSnapshot();
  });
});
