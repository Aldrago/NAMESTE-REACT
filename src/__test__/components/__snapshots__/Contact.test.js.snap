// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact us page test cases Should match snapshort 1`] = `
<div>
  <div>
    <h1
      class="font-bold text-2xl"
    >
      Contact us!
    </h1>
    <form>
      <input
        class=""
        placeholder="name"
        type="text"
      />
      <input
        class=""
        placeholder="message"
        type="text"
      />
      <button>
        Submit
      </button>
    </form>
  </div>
</div>
`;
